templates/_snippets/javascripts.twig

```diff
@@ -59,3 +59,11 @@
         });
     });
 </script>
+
+
+<script type="text/javascript">
+    $('.btn-demo-submit').on('click', function (event) {
+        $(this).prepend('<em class="fas fa-sync-alt fa-spin"></em>&nbsp;&nbsp;&nbsp;');
```

Comment:
Nice one. We could as well disable the input to prevent multiple clicking.

Reply:
I tried many time but failed :cry: 

Feel free to add it :+1: